### ISSUE TITLE
auk_bbox gives Error: length(readLines(x, 2)) not greater than 1

### ISSUE THREAD

**Issue by ajcanepa** [CornellLabofOrnithology/auk#27]
When I'm trying to filter using the bounding box function `auk_bbox()`, always give me the same error:
Error: length(readLines(x, 2)) not greater than 1

I have changed the resolution of the bounding box, the species been filtered (including all species = no filter for species), etc and I always receive the same error message.

The code (not replicable) is:
ebird_data_poligonos <-
  f_in_poligonos %>% 
  # 1. reference file
  auk_ebd() %>% 
  # 2. define filters
  auk_species(species = c("Ciconia ciconia", "Motacilla alba", "Bubulcus ibis")) %>% 
  auk_country(country = "ES") %>% 
  # 3. define the bounding box. Formatted as `c(lng_min, lat_min, lng_max, lat_max)`
  # auk_bbox(bbox = c(42.31, -3.7, 42.37, -3.5)) %>%
  auk_bbox(bbox = c(42.318942, -3.797347, 42.374168, -3.561736)) %>% 
  # 4. run filtering
  auk_filter(file = f_out_poligonos, overwrite = TRUE) %>% 
  # 5. read text file into r data frame
  read_ebd()

Any idea?

**Comment by mstrimas** [comment]
A couple questions:
1. At what point does it give the error? Is it `auk_bbox()`, `auk_filter()`, or `read_ebd()`?
2. Have you tried running the example in the help for `?auk_bbox`? Does that work?
3. Are you using the latest version of `auk` (`remotes::install_github("CornellLabofOrnithology/auk")`) and R, and have you updated all your installed packages?

**Comment by mstrimas** [comment]
@ajcanepa did this problem get resolved?

**Comment by nmanich** [comment]
For what it's worth, I received this error tonight. I wasn't using the bounding box, but I figured out what was going on. I was using a non-original eBird download (I had processed it and reexported as a text file [trying to fix the column header capitalization problem with the current download]) and I ended up giving the new file an initial row number which caused my column numbering on the whole thing to be off. So this was the error it was giving me when only one of the lat long columns was even a number. If you're not using the original eBird download, could be something off with your original input file. Once I fixed this, this error went away.
